src/components/ContactForm.tsx

```diff
@@ -0,0 +1,125 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { FromData } from 'types';
+
+type ContactFormProps = {
+  onSubmit: (formData: FromData) => void;
+};
+
+const ContactForm = ({ onSubmit }: ContactFormProps) => {
+  const [name, setName] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [message, setMessage] = useState<string>('');
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const formData = {
+      name,
+      message,
+    };
+
+    try {
+      console.log(formData);
+      onSubmit(formData);
+      setName('');
```

Comment:
line 23-25 can be combined in one function clearForm(). makes it easier understand what is happening